File: Python/card.py
from numpy import number
from payment import Payment

class card(Payment):
    numberCard = str
    CVV = str 
    dateCard = str

    def __init__(self, id, amount, numberCard, CVV, dateCard):
        super().__init__(id, amount)
        self.numberCard = numberCard
        self.CVV = CVV
        self.dateCard = dateCard
    
    

File: JS/Card.js
class Card extends Payment {
    
    constructor(id, numberCard, CVV, dateCard, amount){
        super(id, amount);
        this.numberCard = numberCard;
        this.CVV = CVV;
        this.dateCard = dateCard;
    }

    printPayment(){
        console.log("Id payment: " +  this.id + " Amount: " + this.amount);
    }
}